server/plugin/webhook.go

```diff
@@ -1045,3 +1049,40 @@ func (p *Plugin) handlePullRequestReviewNotification(event *github.PullRequestRe
 	p.CreateBotDMPost(authorUserID, message, "custom_git_review")
```

Comment:
would it make sense to store the text  `custom_git_review` in a const or enum ?

Reply:
@hanzei your thoughts on this ?

Reply:
Given that the phrase is used in two places, that sounds reasonable to me, but not blocking for the PR. 0/5 defer it to a different one?

Reply:
@hanzei agree, out of scope for this PR.